index.html

```diff
@@ -3221,16 +3250,26 @@ <h3>Reserved Extension Points</h3>
 `https://www.w3.org/2018/credentials#refreshService`.
               </td>
             </tr>
+            <tr>
+              <td>`renderMethod`</td>
+              <td>
+A property used for specifying how to render a credential into a visual,
+auditory, or haptic format. The associated vocabulary URL MUST be
+`https://www.w3.org/2018/credentials#renderMethod`.
+              </td>
+            </tr>
             <tr>
               <td>`termsOfUse`</td>
               <td>
+An example entry for a reserved property.
```

Comment:
Should this be here?

Reply:
No, it shouldn't... bad merge. Fixed in cabcea159857e6f3cfa75cc92cd3d4aed1c0da8f. Thank you!